CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set(script_files
   scripts/termux-api-stop
   scripts/termux-audio-info
   scripts/termux-battery-status
+  scripts/termux-bluetooth-scan
```

Comment:
I think, it would be better name "termux-bluetooth" - without "-scan", or no? It will probably has continuetion as connect or send, am I wrong?

Reply:
Take a look at the rest of this list - there's a lot of `termux-$system-$behavior`. `termux-bluetooth-scan` looks consistent to me.

Reply:
OK, it is true ...